android/2023-emmsale/app/src/main/java/com/emmsale/presentation/ui/notificationBox/recruitmentNotification/recyclerview/body/RecruitmentNotificationBodyClickListener.kt

```diff
@@ -0,0 +1,7 @@
+package com.emmsale.presentation.ui.notificationBox.recruitmentNotification.recyclerview.body
+
+interface RecruitmentNotificationBodyClickListener {
+    fun onRecruitmentAccept(notificationId: Long)
+    fun onRecruitmentReject(notificationId: Long)
```

Comment:
클릭 이벤트를 처리하는 함수이므로
```suggestion
    fun onAcceptButtonClick(notificationId: Long)
    fun onRejectButtonClick(notificationId: Long)
```
이렇게 바꾸면 어떨까요? 혹은 사용자가 모집 수락이나 거절의 이벤트 발생을 추상화(예를 들어 ui를 클릭함으로써 이벤트를 발생시키는 건지 음성인식으로 이벤트를 발생시키는 건지는 감추도록) 하려면 리스너 명을 `RecruitmentNotificationBodyListener`라고 바꾸는 건 어떨까요? 저는 개인적으로 전자를 선호합니다! 안드로이드 의존성을 가진 곳, 즉 UI 로직을 실행하는 클래스에서 사용될 것이기 때문에, ui 이벤트를 추상화하지 않는 게 더 좋다고 생각해요

Reply:
지금 보니, ClickListener 인터페이스에서 onRecruitmentAccept() 라는 네이밍이 조금 어색한 것 같아요.
UI에 대한 Listener인데 비즈니스 로직을 수행할 것 같은..?

해당 리뷰도 반영하도록 하겠습니다 : )